wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -56,6 +57,13 @@ CommandPtr CommandPtr::AsProxy() && {
   return std::move(*this);
 }
 
+CommandPtr CommandPtr::Fork(CommandPtr&& other) && {
```

Comment:
Add an overload for `Fork(CommandPtr&&... other)`

Reply:
I'm getting this error right as I add the overload  
`declaration is incompatible with "frc2::CommandPtr frc2::CommandPtr::Fork(frc2::CommandPtr &&other) &&" (declared at line 60)`

Reply:
Right I guess this makes sense, an overload with a std::span and maybe std::vector works as well

Reply:
like this?

Reply:
That definitely will not work. It would need to be a seperate overload if varargs can work here but I'm not sure they can

Reply:
Okay I've added a separate overload with no major code mods (just a copy).  
I'm getting this error after other

`expected a ')'C/C++(18)
`

Reply:
Which is weird because the parentheses match up.  Is there something i'm missing?

Reply:
> Right I guess this makes sense, an overload with a std::span and maybe std::vector works as well

Sorry I missed this, I'll use std vector

Reply:
I'm sorry, I'm floudering on this.

Reply:
Should I just leave out the overload?